CGFloat values were used as bool values within if statements - this i…

tanzolone opened the pull request:
…s shown as a compiler error on Xcode 7. I have made some changes to check whether those values are == 0 or >0 e.g. (if (value) -> if(value>0, if(!value) -> if(value==0))

Cocoanetics commented:
I have seen quite a few places where you are only comparing > 0, but the value can also be < 0.

Generally any `if (float)` should become `if (float!=0)`

tanzolone commented:
Hi @Cocoanetics - just addressed your comment. I was using `>0` as most of the values are CGFloat sizes that cannot be negative - however there are some offset values which can be indeed be negative. So I have updated the PR to use `!=` instead of `>0` for any `if(float)` in code.

Cocoanetics commented:
Thank you for your efforts!
